.pre-commit-config.yaml

```diff
@@ -107,3 +107,27 @@ repos:
   rev: 2.6.1
   hooks:
     - id: pdm-lock-check
+      name: pdm-lock-check-root
+      entry: pdm lock --check --project .
+      files: ^pyproject.toml$
+- repo: https://github.com/pdm-project/pdm
+  rev: 2.6.1
+  hooks:
+    - id: pdm-lock-check
+      name: pdm-lock-check-server
+      entry: pdm lock --check --project datajunction-server
+      files: ^datajunction-server/pyproject.toml$
+# - repo: https://github.com/pdm-project/pdm
+#   rev: 2.6.1
+#   hooks:
+#     - id: pdm-lock-check
+#       name: pdm-lock-check-query
+#       entry: pdm lock --check --project datajunction-query
```

Comment:
Is there a reason why you kept the datajunction-query one commented out?

Reply:
It will not succeed as the pdm.lock file in that package is out of sync. I wanted to keep the fix for that in a separate change. I want to focus on the fixing the tooling first.